UI/js/common.js

```diff
@@ -0,0 +1,21 @@
+/*
+UI errors for user display
+1000 -> user errors
+*/
+const errors = Object.freeze({
+    //1000 user errors
+    1000: 'Invalid username or password',
+});
+
+const defaultError = 'A system error has occured';
```

Comment:
This does not have to be a system error. we should say something along the lines "Something went wrong, please try again!"

Reply:
agreed